docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,152 @@ directly or used in conditional statements.
+
+    SELECT
+      *
+    FROM
+      TABLE(
+        system.table_changes(
+          schema_name => 'test_schema',
+          table_name => 'tableName',
+          since_version => 0
+        )
+      );
+
+``schema_name`` - type ``VARCHAR``, required, name of the schema for which the function will be called
+
+``table_name`` - type ``VARCHAR``, required, name of the table for which the function will be called
+
+``since_version`` - type ``BIGINT``, optional, version from which changes will be shown, exclusive
+
+Example:
+
+Create table:
+
+.. code-block:: sql
+
+    CREATE TABLE pages (page_url VARCHAR, domain VARCHAR, views INTEGER)
+        WITH (change_data_feed_enabled = true);
+
+Insert some data:
+
+.. code-block:: sql
+
+    INSERT INTO pages
+        VALUES
+            ('url1', 'domain1', 1),
+            ('url2', 'domain2', 2),
+            ('url3', 'domain1', 3);
+    INSERT INTO pages
+        VALUES
+            ('url4', 'domain1', 400),
+            ('url5', 'domain2', 500),
+            ('url6', 'domain3', 2);
+
+Update some data:
+
+.. code-block:: sql
+
+    UPDATE pages SET domain = 'domain4' WHERE views = 2;
+
+Select changes:
+
+.. code-block:: sql
+
+    SELECT
+      *
+    FROM
+      TABLE(
+        system.table_changes(
+          schema_name => 'test_schema',
+          table_name => 'pages',
+          since_version => 1
+        )
+      )
+    ORDER BY _commit_version ASC;
+
+The preceding sequence of SQL statements returns the following result:
+
+.. code-block:: text
+
+  page_url    |     domain     |    views    |    _change_type     |    _commit_version    |    _commit_timestamp
+  url4        |     domain1    |    400      |    insert           |     2                 |    2023-03-10T21:22:23.000+0000
+  url5        |     domain2    |    500      |    insert           |     2                 |    2023-03-10T21:22:23.000+0000
+  url6        |     domain3    |    2        |    insert           |     2                 |    2023-03-10T21:22:23.000+0000
+  url2        |     domain2    |    2        |    update_preimage  |     3                 |    2023-03-10T22:23:24.000+0000
+  url2        |     domain4    |    2        |    update_postimage |     3                 |    2023-03-10T22:23:24.000+0000
+  url6        |     domain3    |    2        |    update_preimage  |     3                 |    2023-03-10T22:23:24.000+0000
+  url6        |     domain4    |    2        |    update_postimage |     3                 |    2023-03-10T22:23:24.000+0000
+
+The output shows what changes happen in which version.
+For example in version 3 two rows were modified, first one changed from
+``('url2', 'domain2', 2)`` into ``('url2', 'domain4', 2)`` and the second from
+``('url6', 'domain2', 2)`` into ``('url6', 'domain4', 2)``.
+
+If ``since_version`` is not provided the function produces change events
+starting from when the table was created.
+
+Example, provided that there is exactly the same table as created above:
```

Comment:
We can omit that .. its implied since we are still talking about the same example